core/Tracker/Response.php

```diff
@@ -161,6 +173,40 @@ private function outputTransparentGif()
         echo base64_decode($transGifBase64);
     }
 
+    /**
+     * Output a custom tracking image
+     *
+     * @param string $customImage The custom image setting specified in the config
+     * @return bool True if the custom image was successfully output, else false
+     */
+    private function outputCustomImage(string $customImage): bool
+    {
+
+        if (strlen($customImage) > 2 && substr($customImage, strlen($customImage)-2, 2) == '==') {
```

Comment:
This should do the same:

```suggestion
        if (strlen($customImage) > 2 && substr($customImage, -2) == '==') {
```